packages/components/organisms/f-footer/CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v4.8.2
+------------------------------
+*February 04, 2021*
+### Changed
+- [CHAD-4892] Updated link lists for AUS Menulog
```

Comment:
Small one – we shouldn't reference JIRA IDs in public repo's (InfoSec don't like us doing it). Can you remove this from the CHANGELOG?

Reply:
Done